unison.opam

```diff
@@ -0,0 +1,23 @@
+opam-version: "2.0"
+maintainer: "juergen@hoetzel.info"
+authors: [
+  "Jürgen Hötzel <juergen@hoetzel.info>"
+  "Louis Gesbert <louis.gesbert@ocamlpro.com>"
+]
+homepage: "https://www.cis.upenn.edu/~bcpierce/unison/"
+bug-reports: "mailto:unison-users@yahoogroups.com "
```

Comment:
yahoogroups has shut down.  see https://github.com/bcpierce00/unison/wiki/Mailing-Lists

Reply:
I pointed to this repository instead. This is usually what people use for this field anyway.
The opam file I used as a base was the latest available version in opam-repository and it contained this.